doc/release-notes.md

```diff
@@ -246,6 +238,42 @@ in the Low-level Changes section below.
 
 - See the [Mining](#mining) section for changes to `getblocktemplate`.
 
+`importmulti` RPC method
+------------------------
+
+Error handling in the `importmulti` RPC method has been much improved. All
+input data is validated before attempting to import the scripts and keys.
+
+Imported scripts/addresses are categorised by solvability and spendability:
+
+- A script is *solvable* if the the wallet can construct the txIn to spend outputs
+  to the script, but cannnot necessarily produce the required signatures for
+  that txIn.
+- A script is *spendable* if the wallet can construct the txIn to spend outputs
```

Comment:
suggested rephrase:

A script is *spendable* if the wallet can construct the txIn to spend outputs with all required signatures or witness data.

Reply:
Changed.